Add article about dns issues at Preply

AmetPreply opened the pull request:
Here is a story how did we get issues with DNS because of conntrack.
https://medium.com/preply-engineering/dns-postmortem-e169efd45afd

hjacobs commented on README.md:
```diff
@@ -3,6 +3,9 @@
 
+* [DNS issues in Kubernetes. Public postmortem #1 - Preply - blog post 2020](https://medium.com/preply-engineering/dns-postmortem-e169efd45afd)
+    * involved: conntrack, DNS, CoreDNS-autoscaler
+    * impact: partial produciton outage
```
small typo

AmetPreply replied:
Nice catch. Thanks!

hjacobs commented:
Thanks!

I have one question: can you provide details on the action item "Patch prod CoreDNS config"?

BTW: You blog post has some links to `localhost` (e.g. "Kibana link") :smirk:

AmetPreply commented:
>BTW: You blog post has some links to localhost (e.g. "Kibana link") 😏

Yes, of course, I can't share a link to the private Kibana dashboard, so I just replaced it with localhost.

>I have one question: can you provide details on the action item "Patch prod CoreDNS config"?

I've changed a

```
Patch prod CoreDNS config
```

to

```
Disable autoscaler for CoreDNS
```
